x-pack/plugins/cases/public/components/all_cases/columns_popover.test.tsx

```diff
@@ -67,7 +66,93 @@ describe('ColumnsPopover', () => {
       expect(onSelectedColumnsChange).toHaveBeenCalledWith([
         { ...selectedColumns[0], isChecked: false },
         selectedColumns[1],
+        selectedColumns[2],
       ]);
     });
   });
+
+  it('clicking Show All calls onSelectedColumnsChange with the right params', async () => {
+    const onSelectedColumnsChange = jest.fn();
+
+    appMockRenderer.render(
+      <ColumnsPopover
+        selectedColumns={selectedColumns}
+        onSelectedColumnsChange={onSelectedColumnsChange}
+      />
+    );
+
+    userEvent.click(await screen.findByTestId('column-selection-popover-button'));
+    userEvent.click(await screen.findByTestId('column-selection-popover-show-all-button'));
+
+    await waitFor(() => {
+      expect(onSelectedColumnsChange).toHaveBeenCalledWith(
+        selectedColumns.map((column) => ({ ...column, isChecked: true }))
```

Comment:
nit: What do you think about moving the `map` outside to avoid calculating the array on each `waitFor` call.